app/GlobalRedux/Feautures/favorites/isFavoritesEmpty-slice.tsx

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { IsFavoritesEmptyState } from '../../../../types/favorites-types';
+import { PayloadAction } from '@reduxjs/toolkit';
+
+const initialState: IsFavoritesEmptyState = {
+  isFavoritesEmpty: true,
+};
+
```

Comment:
I think you don't need create a whole new file for just another reducer state, just put them together.

initialState:{
favoritesItems: [],
isFavoritesEmpty: true,
lastId: 0,
.....
}

Reply:
Done